CoEpi/ui/HealthQuiz/HealthQuizViewController.xib

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You should add constraints to position the elements. 
On my iPhone, the title "Are you experiencing any of the following symptoms?" doesn't show up.
And the table view bg is black, don't know whether it's intended.